docs/maintainer/knowledge_base.md

```diff
@@ -1942,9 +1942,14 @@ There are two options for how to proceed:
 
 <a id="using-centos-7"></a>
```

Comment:
Not sure if we want to touch the anchors here...

Reply:
CC @jaimergp

Reply:
I'd leave it there, since it's still CentOS 7 related-ish, in case someone was linking there somewhere.